### PR TITLE
Fix: kyc forms autofill

### PR DESCRIPTION
**Pull request by josadcha**

### CODE REVIEW

File: src/containers/Confirm/Phone/index.tsx [hunk 1]
```diff
@@ -80,33 +80,38 @@ class PhoneComponent extends React.Component<Props, PhoneState> {
...
-                                    disabled={!phoneNumber}
-                                >
-                                    {this.state.resendCode ? this.translate('page.body.kyc.phone.resend') : this.translate('page.body.kyc.phone.send')}
-                                </Button>
-                            </InputGroup.Append>
-                        </InputGroup>
-                    </fieldset>
+                    <form>
+                        <fieldset className={phoneNumberFocusedClass}>
+                                <InputGroup>
```
Comment (akhlopiachyi) [inline src/containers/Confirm/Phone/index.tsx]: Please, fix indentation

File: src/containers/Confirm/Phone/index.tsx [hunk 2]
```diff
@@ -171,7 +176,7 @@ class PhoneComponent extends React.Component<Props, PhoneState> {
     private handleSendEnterPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
         if (event.key === 'Enter') {
             event.preventDefault();
-            this.handleSendCode();
+            this.handleSendCode(event);
         }
```
Comment (akhlopiachyi) [inline src/containers/Confirm/Phone/index.tsx]: add semicolon

**Comment by pkucherenk0** [comment]
When auto-filling the KYC form there is a border that I would like to not see.
Selection_999(230)

This PR seems like not requested by

**Comment by josadcha** [comment]
Needs 1h to finish

**Comment by josadcha** [comment]
rebased, styles fixed
Screenshot 2020-09-07 at 15 39 30

**Review by oyershov — APPROVED** [review]
LGTM

**Comment by oyershov** [comment]
Better to merge after Q&A

**Pull request merged**